common/data/hmppsAssessmentApi.js

```diff
@@ -61,7 +61,10 @@ const getFilteredReferenceData = (assessmentId, episodeId, questionUuid, parentL
 const getData = (path, tokens) => {
   logger.info(`Calling hmppsAssessments API with GET: ${path}`)
 
-  return action(superagent.get(path), tokens).then(([_, body]) => body)
+  return action(superagent.get(path), tokens).then(([_, body]) => {
+    logger.info(`Calling hmppsAssessments API with GET body: ${body}`)
```

Comment:
Would it be better to have the responsibility of logging in the caller? I'm just wondering if there's a risk in logging the response body here 🤔

Reply:
@LucasCairns I have just seen this, i was using it for testing. You are completely right we shouldn't. I have removed it will create a tiny pr.